### DISCUSSION TITLE
Default value not set for Radio options within a Controller

### DISCUSSION THREAD

**Discussion by oliverhughes** [react-hook-form/react-hook-form#2927] [Q&A] [answered]
Hello!

Amazing library, I'm loving it. But I have ran into an issue, or I am missing something.

When creating a form and passing in default values, I have found that the default Radio input is not initially selected **IF** it is rendered within `Controller`. Radio options controlled via `ref` seem to work fine. 

I have created a dead simple reproduction of the issue here:

https://codesandbox.io/s/react-hooks-form-with-radio-yjysc

Am I missing something? 🤔

**Comment by bluebill1049** [comment, marked as answer]
if you want to build a radio group, take a look at an example from MUI. you need to build a controlled input group: https://codesandbox.io/s/react-hook-form-v6-controller-qsd8r

**Reply by oliverhughes** [reply]
Aha, so it is the controlled input group that is the secret. I have looked at this example already, but perhaps that is what I missed. I'm working with `react-bootstrap` in the project, which does not have an equivalent component. I will have a play and post what I find.

Thanks a lot for the quick reply <3

**Reply by bluebill1049** [reply]
no worries, you can build a wrapper/groupRadio component with react-boostrap for sure.
